Dockerfile-centos7

```diff
@@ -1,6 +1,7 @@
 FROM         centos:7
 MAINTAINER   PcapPlusPlus <pcapplusplus@gmail.com>
 CMD          bash
+ARG          PYTHON_VERSION="3.8.12"
```

Comment:
1. Should we update `CentOS 7` which is legacy?
2. Just curious - is there any particular reason to define this `ARG` other than avoid duplicating `3.8.3` everywhere?

Reply:
ARG is just simplier to read, less error prone so easier to maintain. If we want to bump minor version of python we can do it easily.